### DISCUSSION TITLE
Homematic Accesspoint (NOT CCU)

### DISCUSSION THREAD

**Discussion by MBurchard** [SukramJ/aiohomematic#1516] [Q&A] [answered]
Sorry, I searched the whole evening without luck. Is this Addon for CCU only, or can it be used with the Homematic IP Accesspoint? The Configuration dialog does not tell anything about it.

**Comment by SukramJ** [comment, marked as answer]
Reading the [docs](https://github.com/danielperna84/hahomematic#requirements) is always a great starting point.
